docs/source/guides/building-models/build-config.rst

```diff
@@ -437,54 +437,6 @@ orphaned shapes.
     This transformer does not remove shapes from the prelude.
 
 
-.. _includeAuth-transform:
```

Comment:
These are no longer needed since they're just normal traits. You can include/exclude traits using includeTraits, excludeTraits, etc